Readme.md

```diff
@@ -1,5 +1,15 @@
-# How to resolve a merge conflict
+# How to resolve a merge conflict without losing data
+Additions can sometimes create conflicts even if they do not change any line that another branch had changed.
 In this example project you can demonstrate how to resolve a merge conflict.
+It's important that no useful change is lost during the merge.
+
+# Change branch
+The change branch overwrites a section of the readme with a new line.
+It includes multiple paragraphs of text (which should stay together).
 
 # Goal
```

Comment:
This line and the next was deleted in commit 231cdcf2f1dc44ad7880124e284498314c87ca80. It should not come back after a merge.